### DISCUSSION TITLE
If we have any detail information of the CI in godot Github?

### DISCUSSION THREAD

**Discussion by sanbuphy** [godotengine/godot-proposals#6585] [General Discussions]
Hello!

Recently I have been studying the continuous integration (CI) processes of various open source projects on Github. I noticed that godot has a well-established CI system, and I would like to deepen my understanding of its composition and structure. Could you recommend any relevant materials that I can study and refer to regarding godot 's CI system?

Thank you very much for your help.

**Comment by AThousandShips** [comment]
I'd recommend just studying the code, but please turn to the other [community channels](https://godotengine.org/community) more people there could point you in a good direction, as this is forums is mainly about discussing changes and improvements

**Reply by sanbuphy** [reply]
OK, thanks! I will go there and ask for help.
